src/views/LoginView.vue

```diff
@@ -23,6 +23,10 @@
         type="password"
         placeholder="Password"
       />
+      <div class="remember-me">
```

Comment:
Does this actually do anything yet?

Reply:
Not sure, but this wasn't actually part of the ticket, the only reason it's green is because of rebasing (or some other weird Git behavior)

Reply:
weird ok nvm, i'm sure it's fine